### ISSUE TITLE
[Bug] The "Keep statistics range" option doesn't apply to a single day

### ISSUE THREAD

**Issue by ghost** [Razeeman/Android-SimpleTimeTracker#88]
The "Keep statistics range" option doesn't work if you use these arrows to change between days or if you use the "Select date" option

https://user-images.githubusercontent.com/110909985/224491875-3bee17f4-feae-407a-9d80-4ac82f9a6d8f.mp4

Version: 1.23
Downloaded from F-Droid

**Comment by Razeeman** [comment]
Hello! Thank you for the feedback. It's not specifically a bug because setting state to keep the Range (selected by the middle button) but not a Position (selected by arrow buttons). I don't remember now why it was made like that, but I see the confusion. I'll look into it again and will try to implement it if nothing prevents the position to be also passed to detailed statistics.

**Comment by Razeeman** [comment]
Sorry, you are right. I've checked the code and the Position (which is selected by the arrows) should also be preserved in the detailed statistics. Seems like a bug but I can't reproduce it yet.

**Comment by HelloHumans13** [comment]
Any update on this bug? Facing same here.

**Comment by Razeeman** [comment]
Not really, can't reproduce it unfortunately. Can you share, does it always not work or does it happen from time to time?

**Comment by ghost** [comment]
Whoops, I missed the part in your previous comment where you said that you can't reproduce this bug. For me, it doesn't work every time. Below are the steps to reproduce:

1. Enable "Keep statistic range"
2. Go to the statistics (third tab)
3. Change position by clicking on the arrows, for example, from the present day (today) go to the previous day (yesterday)
4. Click on any activity to view detailed statistics
5. At the bottom, you can see that the app selected present day (today) instead of previous day (yesterday)

I was able to reproduce it on two of my phones, perhaps it only occurs in the F-Droid build?

And as I said in my first comment, this bug also occurs for the "Select date" option.
